Plugin/src/com/n7space/capellaTastePlugin/capellaintegration/EditFunctionActionHandler.java

```diff
@@ -0,0 +1,115 @@
+// N7 Space Sp. z o.o.
+// www.n7space.com
```

Comment:
www -> http:// (www --> 404, https --> bad cert :( )

Reply:
Changed to n7space.com. This works in a browser (unlike www.n7space.com)